Add possibility to enable and test PMU feature

LKomaryanskiy opened the pull request.

lorc commented on meta-xt-domd-gen4/recipes-kernel/linux/linux-renesas/r8a779f0-spider-domd.dts:
```diff
@@ -45,6 +45,14 @@
+	 * as it can not be shared by Xen.
+	 */
+	pmu_a76 {
+		compatible = "arm,cortex-a55-pmu";
```
Do you really need to override `compatible`?

firscity replied:
if so, I think it is better to completely remove existing node (invalid naming) and provide new with correct name and proper `compatible`.

LKomaryanskiy replied:
No, we do not need to override `compatible`, it's a copy-paste issue. I removed this line.

lorc commented on prod-devel-rcar-s4.yaml:
```diff
@@ -79,6 +79,7 @@ common_data:
 
     # Generate ext4 image files
     - [IMAGE_FSTYPES:append, " ext4"]
+    - [IMAGE_INSTALL:append, " perf"]
```
IIRC, we are doing this by image recipe, not via YAML file.

LKomaryanskiy replied:
As I see, we still add the required packages by yaml file:
https://github.com/xen-troops/meta-xt-prod-devel-rcar-gen4/blob/master/prod-devel-rcar-s4.yaml#L150
Or should it should be moved to some other file?

lorc replied:
Ah, okay. Let it be here.

lorc commented on doc/virtualization.md:
```diff
@@ -201,3 +201,11 @@ DHCP. User can provide own IP address by editing
+It's not recommended to use it for production builds
+but it may be  used for development or debugging.
+PMU is disabled by default. It can be enabled by by un-commenting corresponding
+lines in `/etc/xen/domu.cfg` or `/etc/xen/domd.cfg` file in Dom0.
```
Could you please add a couple more paragraphs?
1. Describe why do we need pin vCPUs
2. Describe why it is not securely supported

LKomaryanskiy replied:
Thank you - added descriptions:)

LKomaryanskiy commented:
@lorc, @firscity  thank you for your review comments. I released a new version of PR that includes the following changes:
1. Removed overriding `compatible` field;
2. Added more details about enabling PMU feature.

Please, take a look at the new version.

firscity reviewed (approved):
`Acked-by: Dmytro Firsov <dmytro_firsov@epam.com>`

lorc reviewed (approved):
`Acked-by: Volodymyr Babchuk <volodymyr_babchuk@epam.com>`

LKomaryanskiy commented:
@lorc @firscity thank you for your reviews. Added acked-by tags.
